src/storybook/stand-alone-documentaion/playground/Playground.stories.ts

```diff
@@ -5,4 +5,28 @@ export default {
   decorators: [withPlaygroundRenderer]
 };
 
-export const Playground = {};
+export const Playground = {
```

Comment:
Nice, I liked the initial code 👌🏻. 
Does it still behave well to changes?

Reply:
It works, but if you move between stories, it will always come back to this snippet first. did it behave differently before?

Reply:
It should apply this snippet only on Playground's story

Reply:
cool